api/v1alpha1/pagerdutyintegration_types.go

```diff
@@ -50,6 +50,13 @@ type PagerDutyIntegrationSpec struct {
 
 	// Name and namespace in the target cluster where the secret is synced.
 	TargetSecretRef corev1.SecretReference `json:"targetSecretRef"`
+
+	ServiceOrchestration ServiceOrchestration `json:"serviceOrchestration"`
```

Comment:
Small nit, but can we add description comments to these new structs/fields?

Reply:
Added, thanks!